test/unittests/responses_test.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 let responseModule = require('../../resources/responses.js');
-let Responses = responseModule.responses;
+let Responses = responseModule.Responses;
```

Comment:
Could I ask you to clean this up as you go, and make this `let { Responses, triggerFunctions, severityMap } = require`, here and below? Sorry for not writing that more cleanly in the first place.

Reply:
So, I had another PR planning on cleaning up the imports to use destructuring where things have previously existed. Should I still make that change here or should I just wait until the destructuring PR?

Reply:
Oh, I can wait, then.  :+1: